From c877195f125867a6e6127923b35d9092e20b0929 Mon Sep 17 00:00:00 2001
From: Patrick Domack <patrickdk@patrickdk.com>
Date: Sat, 29 Aug 2015 23:16:58 -0400
Subject: [PATCH] Fix missing now from aprx-stat

---
 aprx-stat.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/aprx-stat.c b/aprx-stat.c
index 7ee75a3..88ac557 100644
--- a/aprx-stat.c
+++ b/aprx-stat.c
@@ -18,6 +18,7 @@ int erlangout;
 int epochtime;
 const char *aprxlogfile;	/* linkage dummy */
 const char *mycall;		/* linkage dummy */
+struct timeval now; // public wall clock that can jump around
 
 #ifdef ERLANGSTORAGE
 
